src/cli/list.rs

```diff
@@ -0,0 +1,283 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use comfy_table::presets::NOTHING;
+use comfy_table::{Attribute, Cell, Color, ContentArrangement, Table};
+use human_bytes::human_bytes;
+use rattler_conda_types::Platform;
+use rattler_lock::{LockedDependency, LockedDependencyKind};
+use serde::Serialize;
+
+use crate::lock_file::load_lock_file;
+use crate::project::SpecType;
+use crate::Project;
+
+// an enum to sort by size or name
+#[derive(clap::ValueEnum, Clone, Debug, Serialize)]
+pub enum SortBy {
```

Comment:
:heart_eyes: Sorting is really cool! Thanks for making this already!

One thing I see would be cool to add is the package type, (`Type`) to sort conda or pypi before one another. But this is a request not a must to get it merged!

Reply:
Sorting by size is super convenient to spot heavy dep!

> One thing I see would be cool to add is the package type, (Type) to sort conda or pypi before one another. But this is a request not a must to get it merged!

I actually did consider it but got lazy. Will do!